src/Object.zig

```diff
@@ -1180,24 +1180,24 @@ pub fn printVersionSections(self: Object, writer: anytype) !void {
             try writer.print("  {x:0>4}", .{count});
 
             for (remaining[0..num]) |versym| {
-                const actual_versym = versym & elf.VERSYM_VERSION;
-                const name = switch (actual_versym) {
-                    elf.VER_NDX_LOCAL => "*local*",
-                    elf.VER_NDX_GLOBAL => "*global*",
+                const actual_versym = versym.VERSION;
+                const name = switch (@as(elf.VER_NDX, @enumFromInt(actual_versym))) {
+                    elf.VER_NDX.LOCAL => "*local*",
+                    elf.VER_NDX.GLOBAL => "*global*",
```

Comment:
FWIW, there's no need to scope the variant for Zig to work out the defining enum, as in, the following is sufficient:
```suggestion
                    .LOCAL => "*local*",
                    .GLOBAL => "*global*",
```